### PR TITLE
Fix WalletConnect V2

### PR DESCRIPTION
**Pull request by AlexBHarley**
### Description

- Upgrade WalletConnect package to beta-18
- Point at wss://relay.walletconnect.org

### Other changes

N/A

### Tested

With a local version of use-contractkit.

### Related issues

N/A

### Backwards compatibility

Not backwards compatible as far as we can see.

### Documentation

N/A

### CODE REVIEW

**Review by gnardini — COMMENTED** [review]
I've tested this and it worked for connecting but when trying to sign I got this error:

Screen Shot 2021-09-27 at 16 16 11

Also, the types added to errors on `catch` statements are causing build issues, we need to remove them.

And one more thing, after syncing with main we need to uncomment a couple lines that were added that disable WC V2 integration on `src/walletConnect/saga.ts`

File: packages/mobile/src/walletConnect/constants.ts [hunk 1]
```diff
@@ -1,38 +1,36 @@
...
+  eth_accounts = 'eth_accounts',
   eth_signTransaction = 'eth_signTransaction',
   eth_sendTransaction = 'eth_sendTransaction',
   eth_signTypedData = 'eth_signTypedData',
   eth_sign = 'eth_sign',
   personal_sign = 'personal_sign',
   personal_decrypt = 'personal_decrypt',
-  // TODO: add back when implemented
-  // computeSharedSecret = 'personal_computeSharedSecret',
+  computeSharedSecret = 'personal_computeSharedSecret',
```
Comment (jeanregisser) [inline packages/mobile/src/walletConnect/constants.ts:11]: Were these implemented?

**Review by jeanregisser — COMMENTED** [review]
Thanks!!

File: packages/mobile/src/walletConnect/constants.ts (hunk 1 above)
Comment (AlexBHarley) [reply packages/mobile/src/walletConnect/constants.ts:11]: No, however we don't do any smart permission overlapping (ie. dapps asks for methods A and B but I only support B so the session is created with just B). My fix was to just enable them for now, if any dApps try and use these methods they'll receive an error.

Comment (AlexBHarley) [reply packages/mobile/src/walletConnect/constants.ts:11]: To be honest I was confused that eth_accounts is there as that's handled on the session handshake for WalletConnect V2.

**Comment by jeanregisser** [comment]
If the PR author has NOT unchecked the box to allow edits by mainainers we can directly push to the branch.

https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork

**Comment by gnardini** [comment]
Oh, I was seeing `Please make sure you have the correct access rights and the repository exists.` but I was clearly doing something else wrong 😅  thanks!

**Pull request merged**